.pre-commit-config.yaml

```diff
@@ -19,13 +19,12 @@ repos:
   hooks:
   - id: isort
     args:
+    # Keep in sync with pyproject.toml
```

Comment:
Also, feedback on writing code comments in general: try to add some context — the minimum and essential amount of information + an external reference for going deeper (if the reader would want to only). This one looks like a TODO item. But it lacks the motivation behind the request — why is it necessary? Does something not work? Does it work differently than one would expect? Is it unintuitive? Should I try to use `git blame` and do some PR/issue archeology each time I see this to figure out if this is still relevant or was even correct in the first place?

Reply:
In fact I am almost sure that isort from precommit just respects the pyproject.toml settings. Maybe @cidrblock forgot to stage current change to the config and was surprised to get different behavior from pre-commit?

Reply:
yep, that must have been it.  comments removed, thanks @ssbarnea @webknjaz (good points about intent in the comments)